python/pyarrow/tests/test_extension_type.py

```diff
@@ -1661,3 +1661,105 @@ def test_legacy_int_type():
             batch = ipc_read_batch(buf)
             assert isinstance(batch.column(0).type, LegacyIntType)
             assert batch.column(0) == ext_arr
+
```

Comment:
```suggestion


```

(2 blank lines between top-level functions. I suppose the linting build will fail about this, but I would recommend to enable the pre-commit checks that also catch some of those issues)